packages/plugin/src/theia.proposed.extensionsAny.d.ts

```diff
@@ -41,9 +41,13 @@ export module '@theia/plugin' {
          * @param extensionId An extension identifier.
          * @param includeDifferentExtensionHosts Include extensions from different extension host
          * @return An extension or `undefined`.
+         *
+         * *Note* In Theia, includeDifferentExtensionHosts will always be set to false, as we only support one host currently.
```

Comment:
That is actually not true: we support at least two different extension hosts: front-end and back-end. Back in the Che days, there was even support for remote-calling the extension interface via RPC.